app/Jobs/PlatformStatsSummaryJob.php

```diff
@@ -29,6 +30,8 @@
 class PlatformStatsSummaryJob extends Job
 {
     private $inactiveThreshold;
+
+    private $platformSummaryStatsVersion = "v1";
     protected $email;
```

Comment:
I guess this field should now be removed?

Reply:
yar!

Reply:
done